whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/UpdateMaintainerSpec.groovy

```diff
@@ -304,6 +323,40 @@ class UpdateMaintainerSpec extends BaseQueryUpdateSpec {
         queryObject("-rGBT mntner CRE-MNT", "mntner", "CRE-MNT")
     }
 
+    def "create maintainer object with invalid  (without -mnt suffix) maintainer name"() {
```

Comment:
Is there a successful test to create a maintainer with an -mnt suffix?

Reply:
Ys, there are existing test that checks the creation of a maintainer and they used -mnt suffix, for example `create self maintained maintainer object check pw hash filtering` and `create maintainer object maintained by other mntner`